res/layout/webxdc_compact_view.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+       xmlns:tools="http://schemas.android.com/tools"
+       xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
thanks, I just copy+pasted from the document and audio views, that should also be removed from there, maybe in a cleanup PR, is the `xmlns:android="http://schemas.android.com/apk/res/android"` needed? I think can be removed as well since it it declared at the top in the parent layout?